src/components/ContentUploader/ContentUploader.js

```diff
@@ -66,6 +66,7 @@ type Props = {
     onComplete: Function,
     onError: Function,
     onMinimize?: Function,
+    onStage: Function,
```

Comment:
Does stage mean staging? If so, wondering if a better name would be onBeforeUpload()?

Reply:
Also instead of Function, can you add the flowtype signature (args:type)=> etc. Starting to add that now instead of just Function.